Source/RuntimeAudioImporter/Private/RuntimeAudioUtilities.cpp

```diff
@@ -81,6 +82,10 @@ void URuntimeAudioUtilities::GetAudioHeaderInfoFromFile(const FString& FilePath,
 			Result.ExecuteIfBound(bSucceeded, MoveTemp(HeaderInfo));
 		}));
 	});
+#else
+	UE_LOG(LogRuntimeAudioImporter, Error, TEXT("file operation support disabled"));
```

Comment:
Could we add a more detailed description here? Such as: `UE_LOG(LogRuntimeAudioImporter, Error, TEXT("Unable to get audio header info from file '%s' because the file operation support is disabled."), *FilePath);`